From: Richard Hansen <rhansen@rhansen.org>
Date: Mon, 23 Mar 2026 21:08:16 -0400
Subject: estimate: Only color the "in NEW as" hint

For consistency with the "see $pkg for inspiration" hint and a planned
"upgrade from $pkg $ver" hint that will be added in the future.
---
 estimate.go | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/estimate.go b/estimate.go
index fa5168b..90e7a72 100644
--- a/estimate.go
+++ b/estimate.go
@@ -352,7 +352,8 @@ func estimate(importpath, revision string) error {
 				if version, ok := sourcesInNew[pkg.source]; ok {
 					url := "https://dfsg-new-queue.debian.org/reviews/" + pkg.source
 					msg := fmt.Sprintf("in NEW as %v %v", pkg.source, version)
-					output(cyanf("%v (%v)", mod, hyperlink(url, msg)))
+					hint += " " + cyanf("(%v)", hyperlink(url, msg))
+					output(mod + hint)
 				}
 				return // already packaged in Debian
 			}
